tests/get_or_create/tests.py

```diff
@@ -503,6 +507,45 @@ def raise_exception():
         )
         self.assertFalse(created)
 
+    def test_mti_update_non_local_concrete_fields(self):
+        journalist = Journalist.objects.create(name="Jane", specialty="Politics")
+        journalist, created = Journalist.objects.update_or_create(
+            pk=journalist.pk,
+            defaults={"name": "John"},
+        )
+        self.assertIs(created, False)
+        self.assertEqual(journalist.name, "John")
+
+    def test_update_only_defaults_and_pre_save_fields_when_local_fields(self):
+        publisher = Publisher.objects.create(name="Acme Publishing")
+        book = Book.objects.create(publisher=publisher, name="The Book of Ed & Fred")
+
+        for defaults, update_time in [
+            ({"publisher": publisher}, datetime(2013, 3, 3, 3, 33, 33, 333333)),
+            ({"publisher_id": publisher}, datetime(2014, 4, 4, 4, 44, 44, 444444)),
+        ]:
+            with self.subTest(defaults=defaults, update_time=update_time):
+
+                with CaptureQueriesContext(connection) as captured_queries:
+                    with freezegun.freeze_time(update_time):
```

Comment:
Would `django.test.utils.freeze_time` be sufficient here?
And therefore we don't need to add `freezegun` 🤞